### PR TITLE
feat(apps): add search bar & misc

### PR DESCRIPTION
**Pull request by kyrie25**
- Adds search bar to App Detection RPC (#279)
- Fix spamming "No updates available" (#279)
- Set Media/Experimental RPC activity state and type on initial start (no buffer time waiting from `Playing` to selected activity type)
- Features will be auto switched off if another feature is turned on to avoid confusion
- Fix clipping title bar text when collapsed
image
- Drawer card layout updates
image
- Minor changes to original strings
- Proofread `vi` strings

### CODE REVIEW

**Comment by kyrie25** [comment]
Everything still works, I just need to customize some stylings to make it fit the overall layout of the app

File: feature_apps_rpc/src/main/java/com/my/kizzy/feature_apps_rpc/AppsRpc.kt [hunk 1]
```diff
@@ -64,12 +72,51 @@ fun AppsRPC(
...
+                                .fillMaxWidth()
+                                .padding(10.dp)
+                        ) {
+                            TextField(
+                                value = searchText,
+                                onValueChange = { searchText = it },
+                                shape = RoundedCornerShape(topStart = 2.dp, topEnd = 2.dp),
+                                modifier = Modifier
+                                    .fillMaxWidth(),
+                                textStyle = MaterialTheme.typography.bodyLarge.copy(color = Color.Black),
```
Comment (dead8309) [inline feature_apps_rpc/src/main/java/com/my/kizzy/feature_apps_rpc/AppsRpc.kt:96]: color shouldn't be set to black explicitly. It doesn't look good in dark theme

Screenshot_20241008-133000_Kizzy

**Comment by dead8309** [comment]
Also you can use [Searchbar](https://github.com/dead8309/Kizzy/blob/25cec8896dc644af281a27113a2172b92f5fbd81/common/components/src/main/java/com/my/kizzy/ui/components/SearchBar.kt) component from common module

File: feature_apps_rpc/src/main/java/com/my/kizzy/feature_apps_rpc/AppsRpc.kt (hunk 1 above)
Comment (kyrie25) [reply feature_apps_rpc/src/main/java/com/my/kizzy/feature_apps_rpc/AppsRpc.kt:96]: That's part of what I was referring to in my comment yes
I'll fix it up when there's time 👍

**Pull request merged**